lib/components/src/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,200 @@
+import React, { Children, useCallback, useEffect, useRef, useState } from 'react';
+import { css, styled } from '@storybook/theming';
+import { AccordionContext } from './AccordionContext';
+
+import type { AddToMapFn, OpenMap } from './AccordionContext';
+
+type AccordionMap = Record<string, { index: number; id: string }>;
+
+type StateChange = {
+  id: string;
+  index: number;
+};
+
+export type AccordionProps = {
+  allowMultipleOpen?: boolean;
+  defaultOpen?: number | number[];
+  bordered?: boolean;
```

Comment:
The border at the last element stands out in both versions:
![image](https://user-images.githubusercontent.com/1671563/120448653-0fa15080-c38c-11eb-8de5-8c1272ed0f70.png)

![image](https://user-images.githubusercontent.com/1671563/120448702-19c34f00-c38c-11eb-959b-f7ecafa431f7.png)

Is that intentional?